reports/institutionFrequentingByUser.jrxml

```diff
@@ -0,0 +1,295 @@
+		<box>
+			<pen lineWidth="0.5" lineColor="#000000"/>
+		</box>
+	</style>
+	<style name="table 1_CH" mode="Opaque" backcolor="#BFE1FF">
+		<box>
+			<pen lineWidth="0.5" lineColor="#000000"/>
+		</box>
+	</style>
+	<style name="table 1_TD" mode="Opaque" backcolor="#FFFFFF">
+		<box>
+			<pen lineWidth="0.5" lineColor="#000000"/>
+		</box>
+	</style>
+	<style name="table 2">
+		<box>
+			<pen lineWidth="1.0" lineColor="#000000"/>
+		</box>
+	</style>
+	<style name="table 2_TH" mode="Opaque" backcolor="#F0F8FF">
+		<box>
+			<pen lineWidth="0.5" lineColor="#000000"/>
+		</box>
+	</style>
+	<style name="table 2_CH" mode="Opaque" backcolor="#BFE1FF">
+		<box>
+			<pen lineWidth="0.5" lineColor="#000000"/>
+		</box>
+	</style>
+	<style name="table 2_TD" mode="Opaque" backcolor="#FFFFFF">
+		<box>
+			<pen lineWidth="0.5" lineColor="#000000"/>
+		</box>
+	</style>
+	<style name="table 3">
+		<box>
+			<pen lineWidth="1.0" lineColor="#000000"/>
+		</box>
+	</style>
+	<style name="table 3_TH" mode="Opaque" backcolor="#F0F8FF">
+		<box>
+			<pen lineWidth="0.5" lineColor="#000000"/>
+		</box>
+	</style>
+	<style name="table 3_CH" mode="Opaque" backcolor="#BFE1FF">
+		<box>
+			<pen lineWidth="0.5" lineColor="#000000"/>
+		</box>
+	</style>
+	<style name="table 3_TD" mode="Opaque" backcolor="#FFFFFF">
+		<box>
+			<pen lineWidth="0.5" lineColor="#000000"/>
+		</box>
+	</style>
+	<subDataset name="dataset1" uuid="c19a8940-5445-4aac-95be-401c7d870329">
+		<parameter name="periods_iso" class="java.lang.String"/>
+		<parameter name="organisationunits_uid" class="java.lang.String"/>
+		<queryString>
+			<![CDATA[SELECT usr.username as User,
+	COUNT(DISTINCT CASE WHEN (p.vih_status = 1 AND DATE_FORMAT(p.date_created, "%Y%m") = $P{periods_iso})
+		THEN p.patient_id else null END) AS 'Nouveau Patient VIH',
+	COUNT(DISTINCT CASE WHEN (p.vih_status = 0 AND DATE_FORMAT(p.date_created, "%Y%m") = $P{periods_iso})
+		THEN p.patient_id else null END) AS 'Nouveau Patient Non VIH',
+	COUNT(DISTINCT CASE WHEN ((p.vih_status = 1 OR p.vih_status = 0 ) AND DATE_FORMAT(p.date_created, "%Y%m") = $P{periods_iso})
+		THEN p.patient_id  else null END) AS 'Nouveau Patient',
+	COUNT(DISTINCT CASE WHEN (p.vih_status = 1 AND DATE_FORMAT(enc.encounter_datetime, "%Y%m") = $P{periods_iso})
+		THEN p.patient_id else null END) AS 'Patient VIH',
+	COUNT(DISTINCT CASE WHEN (p.vih_status = 0 AND DATE_FORMAT(enc.encounter_datetime, "%Y%m") = $P{periods_iso})
+		THEN p.patient_id else null END) AS 'Patient Non VIH',
+	COUNT(DISTINCT CASE WHEN ((p.vih_status = 1 OR p.vih_status = 0 ) AND DATE_FORMAT(enc.encounter_datetime, "%Y%m") = $P{periods_iso})
+		THEN p.patient_id else null END) AS 'Total Patient'
+	FROM openmrs.users usr INNER JOIN isanteplus.patient p ON usr.user_id = p.creator LEFT OUTER JOIN openmrs.encounter enc
+		ON p.patient_id = enc.patient_id, isanteplus.org_code_uid org
+	WHERE ((DATE_FORMAT(enc.encounter_datetime, "%Y%m") = $P{periods_iso})
```

Comment:
I belive this should be
`	WHERE ((DATE_FORMAT(enc.encounter_datetime, "%Y%m") = $P{periods_iso}
		OR (DATE_FORMAT(p.date_created, "%Y%m") = $P{periods_iso})))
		AND p.organisation_uid = org.uid
		AND FIND_IN_SET($P{organisationunits_uid}, org.path)
`

https://stackoverflow.com/questions/1241142/sql-logic-operator-precedence-and-and-or